scripts/readme.md

```diff
@@ -18,6 +18,18 @@ where $PHASE is test, discovery, alpha or beta
 
 Within those directories the definitions of individual fields or registers are in separate YAML files.
 
+This repo also contains files like
+
+    $ROOT/openregister/registry-data/data/$PHASE/$REGISTER/meta.yaml
+
+These contain additional metadata about the register. Each key corresponds to system entries stored in the register, for example "friendly-name".
+
+We intend to keep this github repository in sync with the actual registers until we have a way to archive register data. You can check that the data matches up by running
+
+    OAUTH_TOKEN=<github personal access token> python validate_metadata.py
```

Comment:
can we specify `python3`? As the script did not work for me using `python`.

Reply:
Yep. The prerequisites mentions python3 but I'll update this to be consistent with the other examples